xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm

```diff
@@ -241,7 +241,10 @@ $xwiki.getUserName("xwiki:${username}")
  *   showAlias: false, // whether to show the user alias or not
  *   useInlineHTML: false, // whether to use in-line HTML elements to display the user or not
  *   wrapAvatar: false // whether to wrap the avatar image with a span or not
+ *   displayLink: true // whether to wrap the user name in a link to the user profile
  * }
+ *
+ * @since 15.9
```

Comment:
I don't understand, this macro definitely not been introduced in 15.9 and the introduction of this new option haven't changed its "signature". You might want to document this at the level of the displayLink option instead of here, as it's very missleading.